splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/components/BaseFormView.jsx

```diff
@@ -30,8 +30,10 @@ class BaseFormView extends PureComponent {
                 : `${this.props.serviceName}`;
 
         this.util = {
-            setState: (state) => {
-                this.setState(state);
+            setState: (callback) => {
```

Comment:
If we are removing state from the args then how hook will allow updating state?

Reply:
hook will provide callback function that will receive previous-state as args and will return updated state. 

`this.util.setState((prevState) => {
            return { ...prevState, warningMsg: "This is warning msg" };
          });`